Makefile.include

```diff
@@ -521,3 +521,19 @@ endif
 CFLAGS := $(patsubst -D%,,$(CFLAGS))
 CFLAGS := $(patsubst -U%,,$(CFLAGS))
 CFLAGS += -include '$(RIOTBUILD_CONFIG_HEADER_C)'
+
+# Set a custom channel if needed
+ifneq (,$(filter netdev2_ieee802154,$(USEMODULE)))
```

Comment:
Sorry, I disagree. My point is RIOT should propose an automatic way of setting a DEFAULT_CHANNEL to applications requiring network capabilities. Setting this logic (with the ifneq inside) in a central place, e.g not in the examples folder or in each application Makefile, is the way to go.